features/bootstrap/FixturesContext.php

```diff
@@ -76,6 +86,65 @@ public function weHaveThisTalksInTheSystem(TableNode $table)
             );
 
             $this->getEntityManager()->persist($talk);
+            $event->addTalk($talk);
+        }
+        $this->getEntityManager()->flush();
+    }
+
+    /**
+     * @Given there are no raffles in the system
+     */
+    public function thereAreNoRafflesInTheSystem()
+    {
+        foreach ($this->getRaffleRepository()->findAll() as $raffle) {
+            $this->getEntityManager()->remove($raffle);
+        }
+        $this->getEntityManager()->flush();
+    }
+
+    /**
+     * @Given there are no users in the system
+     */
+    public function thereAreNoUsersInTheSystem()
+    {
+        foreach ($this->getUserRepository()->findAll() as $user) {
+            $this->getEntityManager()->remove($user);
+        }
+        $this->getEntityManager()->flush();
+    }
+
+    /**
+     * @Given we have these users in the system
+     */
+    public function weHaveThisUsersInTheSystem(TableNode $table)
```

Comment:
Typo:
method name should match Behat step definition

weHaveThisUsersInTheSystem > weHaveTheseUsersInTheSystem

Reply:
Ooops, will fix!